components/grid/templates/popup-buttons-template.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Popup Buttons Template
+page_title: Grid Popup Buttons Template
+description: The button template allows you to customize the buttons in the create/edit Popup window of the Grid component.
```

Comment:
When possible, insert the technology in the description of each article (for SEO purposes). 
```suggestion
description: The button template allows you to customize the buttons in the create or edit Popup window of the Blazor Grid component.
```